src/blob_detection.cpp

```diff
@@ -142,6 +142,43 @@ namespace Blob_Detection
     {
         return _contourX.size() == _edgeX.size();
     }
+    
+    double BlobInfo::_getLengthFromContour( const std::vector < uint32_t > & contourX, const std::vector < uint32_t > & contourY, double & length, Point2d & start, Point2d & end ) 
```

Comment:
This function returns `double` by definition but there is no code to do such in function's body.